packages/shared/styles/components/SfArrow.scss

```
@@ -6,9 +6,19 @@ $arrow__button-shadow: 0 5px 12px -6px rgba($c-black, 0) !default;
 $arrow__button-shadow--right: 0 -5px 12px -5px rgba($c-black, 0.6) !default;
 $arrow__icon-color: $c-white !default;
 $arrow__icon-color--hover: $c-dark !default;
+$arrow__icon-height: 12px !default;
```

```suggestion
$arrow__icon-height: 0.75rem !default;
```
if vale is bigger than 10px then use always rem